docs/mesos/DockerizedVagrant.md

```diff
@@ -0,0 +1,130 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying a Vagrant Dockerized Mesos Cluster with Calico
+
+In these instructions, we will create two Centos virtual machines (a master and an agent) that run all cluster services as [Docker][docker] containers.  This speeds deployment and will prevent pesky issues like incompatible dependencies.
```

Comment:
AP  mentioned something to me once that I try to implement in our docs - don't link to any place unless you're telling the user to do something there. This link to Docker doesn't assist users in completing this guide. So just remove it